### ISSUE TITLE
Representing a list of orders

### ISSUE THREAD

**Issue by jchannon** [kevinswiber/siren#71]
If I go to `/orders` typically I would expect that to return a list of orders however the spec says the `properties` item should be a key value pair object so to get around that if I then decide to include each order as sub entities the spec says the rel should identify the sub entity's relation to the parent but in this situation there is no parent.  

What is the best practice to return a list of resources?

**Comment by pmhsfelix** [comment]
Can't we use the "entities" field to include the items representations, using the "item" link relation?

**Comment by kevinswiber** [comment]
Yes.  Per Pedro's advice, this is usually how I represent list items.

**Comment by jchannon** [comment]
Do you keep properties null or add an object with a  count property?

**Comment by kevinswiber** [comment]
Sometimes I toss in a count, as it can be handy for clients, but it depends
on the use case.  The `properties` object is optional, and I prefer
excluding it altogether instead of setting it to `null`.
